scripts/py/meta_hive_species.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+"""
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+'''
+This script takes an directory path as parameter and convert the output from ~metadata/misc_scripts/report_genomes.pl
+and turn it into a flat file names `species_list.txt` to `cat` into a eHive pipeline configuration script.
+Example:
+- species AAA \
+- species BBB \
+     
+'''
+import argparse
+import csv
+import logging
+from os import getenv
+from os.path import isdir
+from os.path import join, isfile, realpath
+
+logging.basicConfig(level=logging.INFO, format='%(message)s')
+
+parser = argparse.ArgumentParser(
+    description='Parse outputs from `~ensemble-metadata/misc_scripts/genomes_report.pl to get species_list for hive'
+                ' init_pipeline command')
+parser.add_argument('-i', '--input_dir', type=str, required=True, help='Input directory')
+parser.add_argument('-d', '--division', help='Restrict to a single division', type=str)
+parser.add_argument('-f', '--output_name', help='Overwrite default output file name', required=False, type=str,
+                    default='species.txt')
+parser.add_argument('-t', '--genomes_types', help='List genomes events (n,r,ua,ug)', required=False, type=str,
+                    default='n,r,ua,ug')
```

Comment:
Unless you want the user to specify the genome types as a comma separated values in a single string, you can use this handy `argparse` argument for treating lists:
```suggestion
parser.add_argument('-t', '--genomes_types', help='List genomes events (n r ua ug)', type=str, nargs='+',
                    default=['n', 'r', 'ua', 'ug'])
```